index.html

```diff
@@ -3186,21 +3186,27 @@ <h3>Terms of Use</h3>
         </p>
 
         <p>
-The value of the <code>termsOfUse</code> <a>property</a> tells the
-<a>verifier</a> what actions it is required to perform (an <em>obligation</em>),
-not allowed to perform (a <em>prohibition</em>), or allowed to perform (a
-<em>permission</em>) if it is to accept the <a>verifiable credential</a> or
-<a>verifiable presentation</a>.
-        </p>
+Examples for the value of the <code>termsOfUse</code> <a>property</a> might be
+to tell the <a>verifier</a> any or all of the following:
+
+<ul><li>
+the procedures or policies that were used in issuing the <a>verifiable credential</a>,
+for example, by providing a pointer to the location where these procedures 
+or policies can be found, or naming the standard that defines them;
+</li>
+<lic
+the issuer's rules and policies that govern the presentation of this 
+<a>verifiable credential</a> to the <a>verifier</a>;
+</li>
+<li>
+the identity of the entity under whose authority the issuer issued this particular <a>verifiable credential</a>.
+</ul></li>
+        </p>
+<p>
+The <code>termsOfUse</code> <a>property</a> SHOULD be automatically processable by the verifier.
 
         <p class="note">
```

Comment:
Given that the content of this `<p class="note">` has been deleted, this `<p class="note">` and the subsequent `</p>` should also be deleted.
```suggestion
```

Reply:
Done. Thanks for spotting it